### PR TITLE
docs(changelog): release notes for v0.19.0

### PR DESCRIPTION
**Pull request by vpavlin**
# Description
 Changelog / Release notes for the v0.19.0

Please review and suggest more/different highlits or notes.

# Changes

- [x] changelog for v0.19.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,58 @@
+## 2023-07-26 v0.19.0
+
+## What's Changed
+
+Release highlights:
+* Postgres support has been added as a backend for archive module
+* RLN initialization optimizations
+* Update to the latest nim-libp2p
```
Comment (jm-clius) [inline CHANGELOG.md:8]: Perhaps a more precise highlight here would be: improved connection management, including management for non-relay peers and limiting the number of connections from a single IP

**Review by jm-clius — APPROVED** [review]
LGTM! Thanks :)

**Review by rymnc — APPROVED** [review]
LGTM

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks for it! :100:

File: CHANGELOG.md (hunk 1 above)
Comment (vpavlin) [reply CHANGELOG.md:9]: Thanks! Added!

**Pull request merged**